packages/wallets/src/evm/connectors/device-wallet/index.ts

```diff
@@ -1,15 +1,16 @@
+import { Chain, normalizeChainId } from "../../../lib/wagmi-core";
 import { ConnectParams, TWConnector } from "../../interfaces/tw-connector";
 import type {
   DeviceWalletConnectionArgs,
   DeviceWalletImpl,
 } from "../../wallets/device-wallet";
-import type { Chain } from "@thirdweb-dev/chains";
 import type { Signer } from "ethers";
 import { providers } from "ethers";
 
 export type DeviceWalletConnectorOptions = {
-  chain: Pick<Chain, "chainId" | "rpc">;
+  chain: Chain;
```

Comment:
Pick didnt work for you here?

Reply:
It _can_ work, but since all wallets are instantiated the same way in `react-core` package, we would need to use that type in all wallets instead. 

<img width="615" alt="image" src="https://user-images.githubusercontent.com/22043396/223651779-382092ca-e006-4144-8d0e-d0398025e11b.png">

Currently this `chain` option is only used by device-wallet. This will be removed soon so that it works similar to other wallets ( provide chain when connecting - not when creating wallet instance )

Now that we are planning to use the `Chain` type from `thirdweb-dev/chains` - we are gonna stop using the `Chain` type from `@wagmi/core` altogether